css/main-css.css

```diff
@@ -49,12 +49,11 @@ h5 {
 
 body{
     width: 100%;
-	background: url('../images/watercolour-header.png') no-repeat center center fixed;
+	background: url('../images/watercolour_header_large.png') no-repeat center center;
```

Comment:
we should actually change the naming convention here

Reply:
No I don't think so